doc/lua_api.txt

```diff
@@ -4864,6 +4864,9 @@ Environment access
 * `minetest.get_objects_inside_radius(pos, radius)`: returns a list of
   ObjectRefs.
     * `radius`: using an euclidean metric
+* `minetest.get_objects_in_area(pos1, pos2)`: returns a list of
+  ObjectRefs.
+     * `pos1` and `pos2` are the min and max positions of the area to search.
```

Comment:
```suggestion
     * `pos1` and `pos2` are the corner positions of the volume to search.
```

Reply:
`* `minetest.find_nodes_in_area(pos1, pos2, nodenames, [grouped])`
    * `pos1` and `pos2` are the min and max positions of the area to search.`

Reply:
I just copied the description of `find_nodes_in_area`.